src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      data-qa="big"
+      class="data"
+    >
+      <label class="container">
+        <input
+          type="text"
+          name="search-big"
+          data-qa="keypress"
+          placeholder="Try &quot;Los Angeles&quot;"
+          class="search big"
+        >
+        <img
+          class="icon icon-big"
+          src="images/Search.svg"
+          alt="search"
+        >
+      </label>
+    </form>
+    <form
+      action="#"
+      data-qa="small"
+      class="data"
+    >
+      <label class="container">
+        <input
+          type="text"
+          name="search-small"
+          data-qa="keypress"
+          placeholder="Try &quot;Los Angeles&quot;"
+          class="search small"
+        >
+        <img
+          class="icon icon-small"
+          src="images/Search.svg"
+          alt="search"
+        >
+      </label>
+    </form>
   </body>
```

Comment:
```suggestion
    <form
      action="#"
      data-qa="big"
      class="form"
    >
      <label class="container">
        <input
          type="text"
          name="search-big"
          data-qa="keypress"
          placeholder="Try &quot;Los Angeles&quot;"
          class="search"
        >
        <img
          class="icon"
          src="images/Search.svg"
          alt="search"
        >
      </label>
    </form>
    <form
      action="#"
      data-qa="small"
      class="form"
    >
      <label class="container">
        <input
          type="text"
          name="search-small"
          data-qa="keypress"
          placeholder="Try &quot;Los Angeles&quot;"
          class="search small"
        >
        <img
          class="icon icon-small"
          src="images/Search.svg"
          alt="search"
        >
      </label>
    </form>
  </body>
```
lets make it work like this